docs/versioned_docs/version-v1.3.0/02-Features/02-message-handling/03-event-hubs.md

```diff
@@ -0,0 +1,319 @@
+﻿---
+title: "Azure Event Hubs message handling"
+layout: default
+---
+
+# Azure Event Hubs message handling
+The `Arcus.Messaging.Pumps.EventHubs` library provides ways to perform all the plumbing that is required for processing event messages on Azure EventHubs. 
+
+As a user, the only thing you have to do is **focus on processing messages, not how to get them**. Following terms are used:
+- **Message handler**: implementation that processes the received message from an Azure EventHubs. Message handlers are created by implementing the `IAzureEventHubsMessageHandler<TMessage>`. This message handler will be called upon when a message is available in the Azure EventHubs. [this section](#message-handler-example) for a message handler example setup
+- **Message router**: implementation that delegates the received Azure EventHubs event message to the correct message handler.
+- **Message pump**: implementation that interacts and receives the Azure EventHubs event message. The pump can be configured for different scenarios, see [this section](#pump-configuration) for more information.
+
+![Message handling schema](/media/worker-eventhubs-message-handling.png)
+
+## Installation
+This features requires to install our NuGet package:
+
+```shell
+PM > Install-Package Arcus.Messaging.Pumps.EventHubs
+```
+
+## Message handler example
+Here is an example of a message handler that expects messages of type `Order`:
```

Comment:
I'm now nitpicking, but you typically do not use an EventHub to send messages that represent an `Order`, or something that has 'business-value'.
Maybe you should modify the example so that you process messages that represent telemetry.  For instance `SensorReading` or `DeviceTelemetry` or something like that ...

Reply:
It's a pity that this wasn't fleshed out in the PR's about EventHubs.
I'll see if we can change this.

Reply:
https://github.com/arcus-azure/arcus.messaging/pull/337/commits/f5763a9949016e4d29c401337af5a26c2f3f0eba